src/odemis/gui/cont/project.py

```diff
@@ -468,8 +467,8 @@ def __init__(self, tab_data, calibration_panel, viewport, calib_prefix):
         # Only enable buttons for scintillators which have been selected in the chamber tab
         tab_data.main.active_scintillators.subscribe(self._on_active_scintillators)
 
-        tab_data.main.is_acquiring.subscribe(self._on_is_acquiring)  # enable/disable calib button during acquisition
-        tab_data.is_calibrating.subscribe(self._on_is_acquiring)  # enable/disable calib button during calibration
+        tab_data.main.is_acquiring.subscribe(self._on_is_calibrating)  # enable/disable calib button during acquisition
```

Comment:
shouldn't this be `self._on_is_acquiring`? Or is the comment incorrect?

Reply:
Here, we check whether an acquisition (`is_acquiring`) or a calibration (`is_calibrating`) is ongoing. If so, in both cases `_on_is_calibrating` is called and the calibration panels get enabled or disabled depending on the state (`is_acquiring`/`is_calibrating`). The acquisition panel (button to acquire ROAs) is handled elsewhere.